src/radical/entk/task.py

```diff
@@ -991,6 +1020,7 @@ def to_dict(self):
 
             'exit_code'            : self._exit_code,
             'path'                 : self._path,
+            'tags'                 : self._tags,
             'tag'                  : self._tag,
```

Comment:
Then `self._tag` could be removed altogether...

Reply:
Yes.